data/config.ini

```diff
@@ -0,0 +1,11 @@
+[geo]
+OSMFile = ghana-internal.osh.pbf
+MapCenter = 5.6204/-0.2125
+
+[temporal]
+StartDate = 2017-07-05
+EndDate = 2017-09-05
+DeltaDays = 10
```

Comment:
It was a Delta of 1 in the source code. Why changing to 10 here?

Reply:
my bad, it was to test quicker.